### ISSUE TITLE
V5.2 - Delete contact confirm button german translation character error

### ISSUE THREAD

**Issue by stefan-franz** [mautic/mautic#14316]
### Mautic Series

5.1.x series

### Mautic installed version

5.2

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.2.26

### What browsers are you seeing the problem on?

Firefox

### What happened?

German translation for delete is "löschen" or "Löschen"
It is displayed right, except in this confirm dialogue e.g. if i want to delete a contact

Here it's displayed right
![Löschen](https://github.com/user-attachments/assets/5560c3a8-4b47-418e-aec3-f2fcebd156aa)

Here is a character error
![Kontakt löschen](https://github.com/user-attachments/assets/4084f570-4379-43f9-bf0c-d5392fb393ad)

My question: In which translation file is the translation for this confirm button? 

I searched here and "Löschen" (capital letter at the beginnung) is translated right. 
In my optinion its a problem in Mautic with the "ö" in Löschen

Searched for "Löschen"
`xxxxx@yyyy:/var/www/html/mautic/translations/de# grep -r 'Löschen'
CoreBundle/messages.ini:mautic.core.form.confirmbatchdelete="Löschen der gewählten Elemente?"
CoreBundle/messages.ini:mautic.core.form.delete="Löschen"
CoreBundle/messages.ini:mautic.keyboard.delete="Löschen"
CoreBundle/messages.ini:mautic.core.permissions.delete="Löschen"
DynamicContentBundle/messages.ini:mautic.dynamicContent.form.confirmdelete="Löschen Sie das Element mit dynamischem Inhalt, WARNUNG - dadurch werden auch alle zugehörigen Übersetzungen gelöscht!"
MauticFocusBundle/messages.ini:mautic.focus.form.confirmbatchdelete = "Löschen der ausgewählten Focus Items?"
root@mautic01:/var/www/html/mautic/translations/de# 
`

### How can we reproduce this issue?

Switch to german Language or tell me, what i can check to find the error (if i'm the only)

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by matbcvo** [comment]
Thanks for reporting the bug, @stefan-franz 
However, this appears to be a duplicate of an issue that has already been reported.
Please refer to https://github.com/mautic/mautic/issues/13706 for a detailed overview of the issue and the steps taken to resolve it.

**Comment by stefan-franz** [comment]
If you don't want to fix it - i can live with it. In the issue you refered, i can't see any solutions. And i reportet, that in V5.2 is the same error. 
I described the problem and gave all infos i had and you close it without any thinking of it. It would more useful in my opinion you close the old ticket your refered.

**Comment by matbcvo** [comment]
Thank you for your feedback and for reporting this issue.

I closed this ticket because it is a duplicate of an issue already logged, which helps centralize discussions and ensures all relevant information is in one place. I understand your concern about not seeing a solution in the linked issue, but the goal is to avoid splitting discussions across multiple tickets to ensure all relevant information is centralized and more easily tracked by the development team.

I’ve noted that this issue also occurs in versions 5.2, 5.1.x, and 5.0.x. Since this ticket is referenced in the original issue, your report is linked and visible there for further context. While it hasn’t been picked up for a fix yet, you’re welcome to contribute by working on it and submitting a pull request. Your detailed description and insights are greatly appreciated.

Thank you for your understanding!
